assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -166,13 +167,23 @@ function Carousel() {
     <>
       <Wrapper>
         <Area area="left-navigation">
-          <LeftArrow
-            isHidden={!hasHorizontalOverflow || isAtBeginningOfList}
-            onClick={() => scrollBy(-(2 * PAGE_THUMB_WIDTH))}
-            width="24"
-            height="24"
-            aria-label={__('Scroll Left', 'web-stories')}
-          />
+          {isRTL ? (
```

Comment:
I feel like flexbox's direction and order properties could offer some solutions for us. Alternatively we could rename `left/right-navigation` appropriately to `next/prev` and shuffle them around completely in CSS. Perhaps styled has some good hook ups for that.

Also, does this mean that we need to do something with scroll direction as well?

Reply:
Renaming definitely makes sense 👍 

And yes, scroll direction is changed here as well. But perhaps I can work around it. Will test again.

Reply:
So, we're using CSS Grid here for the layout, which is nicely flipped for us automatically when `direction: rtl`, so the layout is not an issue.

However, when the Prev Button is now suddenly flipped, we need to use the RightArrow icon instead of LeftArrow. And scrolling back now means scrolling all the way to the right instead of the left.

I've updated the carousel and pagenav components now in a way that I think makes most sense.

Reply:
Yeah, this one in particular is an interesting case. We don't have to endow these with the semantics - we can think of these as just a scrolling logic.